zuul.d/content_provider.yaml

```diff
@@ -16,18 +16,35 @@
       - ^kuttl-test.yaml$
     required-projects:
       - opendev.org/zuul/zuul-jobs
+      - openstack-k8s-operators/ansible-microshift-role
+      - openstack-k8s-operators/ci-framework
+      - openstack-k8s-operators/cinder-operator
+      - openstack-k8s-operators/data-plane-adoption
+      - openstack-k8s-operators/dataplane-operator
+      - openstack-k8s-operators/designate-operator
+      - openstack-k8s-operators/edpm-ansible
+      - openstack-k8s-operators/edpm-image-builder
+      - openstack-k8s-operators/glance-operator
+      - openstack-k8s-operators/heat-operator
+      - openstack-k8s-operators/horizon-operator
+      - openstack-k8s-operators/infra-operator
       - openstack-k8s-operators/install_yamls
-      - openstack-k8s-operators/openstack-operator
-      - github.com/openstack-k8s-operators/ci-framework
-      - github.com/openstack-k8s-operators/repo-setup
-      - openstack-k8s-operators/openstack-ansibleee-operator
+      - openstack-k8s-operators/ironic-operator
       - openstack-k8s-operators/keystone-operator
-      - openstack-k8s-operators/nova-operator
+      - openstack-k8s-operators/lib-common
+      - openstack-k8s-operators/manila-operator
       - openstack-k8s-operators/mariadb-operator
-      - openstack-k8s-operators/dataplane-operator
-      - openstack-k8s-operators/placement-operator
+      - openstack-k8s-operators/neutron-operator
+      - openstack-k8s-operators/nova-operator
+      - openstack-k8s-operators/octavia-operator
+      - openstack-k8s-operators/openstack-ansibleee-operator
       - openstack-k8s-operators/openstack-baremetal-operator
+      - openstack-k8s-operators/openstack-operator
+      - openstack-k8s-operators/os-diff
       - openstack-k8s-operators/ovn-operator
+      - openstack-k8s-operators/placement-operator
+      - openstack-k8s-operators/repo-setup
+      - openstack-k8s-operators/tcib
```

Comment:
The above list looks good, @viroel Can we install SF zuul github app on barbican and swift operator? and add it to the list.